abd-clam/src/core/cluster/mod.rs

```diff
@@ -10,6 +10,7 @@ mod _cluster;
 mod criteria;
 mod tree;
 
-pub use _cluster::Cluster;
+#[allow(clippy::module_name_repetitions)]
+pub use _cluster::{Cluster, SerializedChildInfo, SerializedCluster};
```

Comment:
We will likely end up making `SerializedChildInfo` private within the `_cluster` module.